assets/data/plugin_denylist.txt

```diff
@@ -33,4 +33,4 @@ wpmandrill
 yotpo
 zapier
 zoho-mail
-
```

Comment:
Can you remove this blank line? Also while you're editing this file, can you sort them alphabetically? That will make it a little easier to check if a plugin is already listed or not. So it should be:

console
mail-bank
mailster
metorik
paypal
postmark
postmatic
referralcandy
sendgrid
sendinblue
shareasale
shipstation
smtp
socketlabs
sparkpost
stripe
webgility
woocommerce-zapier
wp-remote-users-sync
wp-ses
wpmandrill
yotpo
zapier
zoho-mail